### DISCUSSION TITLE
Changelog notification

### DISCUSSION THREAD

**Discussion by Deekerman** [tteck/Proxmox#2094] [Q&A]
So.... I basically look every morning to see what new script is available from tteck. Is there a way to get notifications anytime you post a new script? its possible there is something within github but as I have tried to make abundantly clear, i am not great at this...not even a little. 

Anyways, could you make a notification system for when you post new things?

**Comment by OddSquirrel** [comment]
Not tteck, but why not grab one of his very scripts (Change Detection LXC) and enjoy notifications without much fuss?

**Reply by Deekerman** [reply]
i tried that once, and probably due to a lack of understanding of how to make it work, my inbox blew up with any change to the site.  but i can revisit. 

-edit- it might be different now that there are no numbers on the tteck site related to the commits he has on git

**Reply by danbayliss** [reply]
Try https://visualping.io

**Comment by oOStroudyOo** [comment]
I've only just installed Change Detection and guessed my way through it.
I won't be able to tell if it works until Tteck adds a new script but this is what I've done:

- Installed change detection via `https://helper-scripts.com/scripts?id=Change+Detection`
- Opened change detection in my web browser
- Under add new change detection watch, inserted `https://helper-scripts.com/scripts` in the URL field and left Webpage Text ticked.
- Click Edit > Watch
- Gave it a title (`New Tteck Scripts`)
- Selected how often I wanted the check to happen (Default is every 3 hours).
- In the Request tab, I selected `Playwright Chromium`
- In the Visual Filter Selector, I grab the section that I wanted to detect a change on, which was the first slot under the Newest Scripts header.
- It's quite hard to select so go to Filters & Triggers and paste in `/html/body/div/div[2]/div/div[2]/div/div[2]/div/div[1]/div/div[1]` under CSS Filters if you wish to use the same as me.
- I added my preferred notifications on the next tab which is a Discord intergration.
- Pressed Save.

**Reply by oOStroudyOo** [reply]
I need to do some tidying up on the notification, but the function works fine:
![image](https://github.com/tteck/Proxmox/assets/111181956/32518c4e-b6e7-4b52-9888-9b0c6392392f)
